apidoc/Titanium/Android/Notification.yml

```diff
@@ -366,9 +366,44 @@ properties:
     since: "3.6.0"
     osver: {android: {min: "5.0"}}
 
+  - name: wakeLock
+    summary: |
+        Will wake up the device for the given time (in milliseconds) when the notification is shown.
+        The application needs to also set the `android.permission.WAKE_LOCK` permission
+        in the Android manifest section of the `tiapp.xml` file.
+        
+        <ti:app>
+            <android>
+                <manifest>
+                    <uses-permission android:name="android.permission.WAKE_LOCK" />
+                </manifest>
+            </android>
+        </ti:app>
+    type: wakeLockOptions
+    default: 0
+    since: "6.2.0"
+    
   - name: when
     summary: The timestamp for the notification (defaults to the current time).
     description: |
         If the timestamp is set to 0, no time is displayed in the notification
         window for this notification.
     type: [Date,Number]
+
+---
+name: wakeLockOptions
+summary: Optional parameter to enable animation to [hide](Titanium.UI.View.hide) and [show](Titanium.UI.View.show).
+since: 6.2.0
+properties:
+  - name: time
+    summary: Minimum time the device will be switched on (plus device switch-off time)
+    type: int
+    default: 3000
+  - name: flags
+    summary: Wake lock level and flag. See [Android Developers: PowerManager](https://developer.android.com/reference/android/os/PowerManager.html#newWakeLock(int,%20java.lang.String))
+    type: int
+    constants: |
+        Titanium.Android.PARTIAL_WAKE_LOCK | Titanium.Android.FULL_WAKE_LOCK | 
+        Titanium.Android.SCREEN_DIM_WAKE_LOCK | Titanium.Android.SCREEN_BRIGHT_WAKE_LOCK
+        Titanium.Android.ACQUIRE_CAUSES_WAKEUP | Titanium.Android.ON_AFTER_RELEASE
+    default: Titanium.Android.FULL_WAKE_LOCK | Titanium.Android.ACQUIRE_CAUSES_WAKEUP | Titanium.Android.ON_AFTER_RELEASE
```

Comment:
We usually prefix the constants so it's easier to find and group them. For example:
- WAKE_LOCK_PARTIAL
- WAKE_LOCK_FULL
- WAKE_LOCK_SCREEN_DIM

Then you could also also write `constants: [Titanium.Android_WAKE_LOCK_*]`